### PR TITLE
feat: add login and signup page

### PR DESCRIPTION
**Pull request by Prathyusha-96**
image
image

### CODE REVIEW

File: pages/login.css [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+}
+.heading {
+    margin-bottom: 2rem;
+    padding: 2rem 2rem 0 2rem;
+}
+.text-center {
+    text-align: center;
+}
+.input {
+    color: #333;
```
Comment (RakshaPawar108) [inline pages/login.css]: You can make use of a variable here. Create one for it in the root just like you have done for text color

File: pages/login.css [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+    margin: 0 1.1rem 0.8rem;
+    width: 80%;
+}
+label {
+    display: block;
+    margin-bottom: 2px;
+}
+
+.input-txt {
+    background: #fff;
```
Comment (RakshaPawar108) [inline pages/login.css]: I would recommend using variables wherever you can for colors at least as you have used direct hex codes in many places in your code.

File: pages/login.html [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="login.css">
+</head>
+<body>
+    <div class="container">
```
Comment (RakshaPawar108) [inline pages/login.html]: You can make use of semantic HTML tags like `main`, `section`, `aside`, `header`, `footer`, and so on for main parts of your code for accessibility and better code readability. You can use div tags for smaller chunks of code inside the main sections.

**Review by RakshaPawar108 — CHANGES_REQUESTED** [review]

**Review by Aazam10 — APPROVED** [review]
Great Work.  One suggestion you can add for attribute in label and give that same to id of input this will help when adding logic.

**Review by mohammed-sas — APPROVED** [review]
suggestion: it's good to have a navbar or navbrand on the top/ corner so that user would know on which site they're at.
pr good to merge

**Review by RakshaPawar108 — APPROVED** [review]
Looks good to merge. And even I would agree with Mohammed's comment to add a navbar to your page to let the user know which site they are on.

**Pull request merged**